Neos.Neos/Resources/Private/Fusion/RawContent/Node.fusion

```diff
@@ -0,0 +1,45 @@
+
+prototype(Neos.Neos:RawContent.Node) < prototype(Neos.Neos:ContentComponent) {
+    node = null
+
+    renderer = Neos.Fusion:Case {
+
+        custom {
+            @position = "100"
+            condition = Neos.Fusion:CanRender {
+                type = ${props.node.nodeType.name + '.RawContentMode'}
+            }
+            renderer = Neos.Fusion:Renderer {
+                @context.node = ${props.node}
+                type = ${props.node.nodeType.name + '.RawContentMode'}
+                element.node = ${props.node}
```

Comment:
i dont see the value of saving `node = ${node}` in the component ... and injecting node magically. `@context` is for most people scary enough already this could blow them away xD